packages/gatsby-transformer-sharp/src/gatsby-node.js

```diff
@@ -1,32 +1,14 @@
 const fs = require(`fs-extra`)
 
 exports.onCreateNode = require(`./on-node-create`)
-exports.setFieldsOnGraphQLNodeType = require(`./extend-node-type`)
+exports.createSchemaCustomization = require(`./customize-schema`)
```

Comment:
Need to bump dependency on `gatsby` since older versions won't have this API.

Reply:
Added in https://github.com/gatsbyjs/gatsby/pull/18871/commits/2fb1c2061f52ef0b934c8fd6c5e3b7fb52c05406